skt/reporter.py

```diff
@@ -474,13 +475,13 @@ class stdioreporter(reporter):
 
     def report(self):
         self.update_mergedata()
-        print "Subject: %s\n" % self.getsubject()
-        print self.getreport()
+        print("Subject: ", self.getsubject(), sep='')
```

Comment:
Thanks for getting rid of the newlines, but I think you misunderstood the space removal here. Separator has effect on the all arguments so if we wanted to append something to the subject, we'd need to accommodate for it. Removing the space (instead of adding the separator) is less error-prone for that.